<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeWidget</class>
 <widget class="QWidget" name="TreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0" columnstretch="1,0">
   <item row="1" column="1">
    <layout class="QGridLayout" name="gl_chart"/>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cb_onlyDirect">
     <property name="text">
      <string>Show only direct children</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_kind">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lbl_txt_name">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_type">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lbl_txt_kind">
        <property name="text">
         <string>Kind:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lbl_txt_type">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_timetaken">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lbl_txt_timetaken">
        <property name="text">
         <string>Time taken:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lbl_txt_timetakenratio">
        <property name="text">
         <string>Time taken ratio:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lbl_timetakenratio">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeWidget" name="tw_tree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <column>
      <property name="text">
       <string notr="true">Name</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
